sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -372,6 +386,59 @@ case class GpuCast(
       case (ShortType | IntegerType | LongType | ByteType | StringType, BinaryType) =>
         input.getBase.asByteList(true)
 
+      case (IntegerType | LongType, dt: DecimalType) =>
```

Comment:
What about byte and short which are also listed as supported for ANSI cast in the docs?

Reply:
Sorry this is still missing byte.  Is that expected?